pkg/utils/unstructerd_wrappers.go

```diff
@@ -0,0 +1,31 @@
+package utils
+
+import (
+	"fmt"
+
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+)
+
+func NestedStringWrapper(obj map[string]interface{}, fields ...string) (string, error) {
+	s, ok, err := unstructured.NestedString(obj, fields...)
+	if !ok {
+		return "", fmt.Errorf("can't find %s", fields)
```

Comment:
I don't know how, but it works. I thought `%s` had to be a string, but apparently a slice is fine too. I don't think I've seen it used like this before instead of `%v`.

Reply:
Good point. It's true that it looks little bit weird e.g `can't find [spec displayName]`....I think I will try to update. Thanks